edgexfoundry/device-usb-camera
# refactor!: Change configuration and device toml files to yaml

BREAKING CHANGE: Configuration and device files now use yaml







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
1. start up edgex with no-secty
2. `make build`
3. run `./device-usb-camera -cp -o`
4. verify device-usb-camera  configuration loaded with no error

## New Dependency Instructions (If applicable)
